### PR TITLE
UI adjusments to harmonize components and fix some visual bugs

### PR DESCRIPTION
**Pull request by klesaulnier**

### CODE REVIEW

File: src/components/dialogs/create-case-dialog.js [hunk 1]
```diff
@@ -88,10 +93,7 @@ export function CreateCaseDialog({ onClose, open }) {
                 <FormattedMessage id="ImportNewCase" />
             </DialogTitle>
             <DialogContent>
-                <DialogContentText>
-                    <FormattedMessage id="createNewCaseDescription" />
-                </DialogContentText>
-                <div>{nameField}</div>
+                {nameField}
```
Comment (Meklo) [inline src/components/dialogs/create-case-dialog.js]: Since nameField seems to be a component, wouldn't it be logical to capitalise it ?

**Review by Meklo — APPROVED** [review]

File: src/translations/en.json [hunk 2]
```diff
@@ -101,11 +98,10 @@
     "renameDirectoryError": "You cannot rename a directory you don't own",
     "edit": "Edit",
 
-    "createNewContingencyList": "Create new contingency list",
-    "createNewContingencyListDescription": "To create a new contingency list, you should enter the following information:",
+    "createNewContingencyList": "Create contingency list",
```
Comment (sBouzols) [inline src/translations/en.json]: Create **a** contingency list ?

File: src/translations/en.json [hunk 3]
```diff
@@ -10,8 +10,7 @@
     "public": "public",
     "emptyDir": "Empty directory",
 
-    "createNewStudy": "Create new study",
-    "createNewStudyDescription": "To create a new study, you should enter the following information:",
+    "createNewStudy": "Create study",
```
Comment (sBouzols) [inline src/translations/en.json]: Create **a** study ?

File: src/translations/en.json [hunk 4]
```diff
@@ -23,10 +22,9 @@
     "deleteStudyError": "You are not the owner of this study",
     "deleteElementsFailure": "Problem encountered while deleting {stn, plural, one {an element} other {elements}}. Not deleted : {problematic}",
     "moveElementsFailure": "Problem encountered while moving {stn, plural, one {an element} other {elements}}. Not moved : {problematic}",
-    "caseExist": "Does the case already exist?",
-    "studyName": "study name",
+    "studyName": "name",
```
Comment (sBouzols) [inline src/translations/en.json]: then the key should be "Name" and could be used everywhere without duplication ?

File: src/translations/en.json [hunk 5]
```diff
@@ -23,10 +22,9 @@
...
     "studyNameProperty": "Study name: ",
-    "studyDescription": "study description",
+    "studyDescription": "description (optional)",
```
Comment (sBouzols) [inline src/translations/en.json]: then the key should be "DescriptionOpt" and could be used everywhere without duplication ?

File: src/translations/en.json [hunk 6]
```diff
@@ -101,11 +98,10 @@
     "renameDirectoryError": "You cannot rename a directory you don't own",
     "edit": "Edit",
 
-    "createNewContingencyList": "Create new contingency list",
-    "createNewContingencyListDescription": "To create a new contingency list, you should enter the following information:",
+    "createNewContingencyList": "Create contingency list",
     "elementNameErrorMsg": "The element name is not provided",
-    "contingencyListName": "Name of contingency list",
-    "contingencyListDescription": "Contingency list description",
+    "contingencyListName": "name",
```
Comment (sBouzols) [inline src/translations/en.json]: Name with N uppercase ?

File: src/translations/en.json [hunk 7]
```diff
@@ -101,11 +98,10 @@
...
+    "contingencyListDescription": "description (optional)",
```
Comment (sBouzols) [inline src/translations/en.json]: **D**escription with D uppercase ?

File: src/translations/fr.json [hunk 8]
```diff
@@ -23,10 +22,9 @@
     "deleteStudyError": "Vous n'êtes pas le propriétaire de l'étude",
     "deleteElementsFailure": "Problème à la destruction {stn, plural, one {d'un élément} other {d'éléments}}. Pas {pbn, plural, one {détruit} other {détruits}} : {problematic}",
     "moveElementsFailure": "Problème au déplacement {stn, plural, one {d'un élément} other {d'éléments}}. Pas {pbn, plural, one {déplacé} other {déplacés}} : {problematic}",
-    "caseExist": "La situation existe-t-elle déjà ?",
-    "studyName": "nom de l'étude",
+    "studyName": "nom",
```
Comment (sBouzols) [inline src/translations/fr.json]: **N**om with N uppercase

Comment (sBouzols) [reply src/translations/fr.json]: and same remarks as for en.json file...

File: src/translations/fr.json [hunk 9]
```diff
@@ -23,10 +22,9 @@
...
     "studyNameProperty": "Nom de l'étude : ",
-    "studyDescription": "description de l'étude",
+    "studyDescription": "description (optionnel)",
```
Comment (sBouzols) [inline src/translations/fr.json]: optionnelle ?

File: src/components/dialogs/create-filter-dialog.js [hunk 10]
```diff
@@ -239,57 +238,54 @@ const CreateFilterDialog = ({
...
-                            />
-                            <FormControlLabel
-                                value="FORM"
-                                control={<Radio />}
-                                label={<FormattedMessage id="FORM" />}
-                            />
-                        </RadioGroup>
-                    </Grid>
-                </Grid>
+            <DialogTitle onClose={handleClose}>{title}</DialogTitle>
```
Comment (sBouzols) [inline src/components/dialogs/create-filter-dialog.js:247]: if you don't use `DialogContainer`, `CustomDialogTitle`, `CustomDialogContent`  or `CustomDialogActions` anymore, remove it if not used elsewhere

File: src/components/dialogs/create-directory-dialog.js [hunk 11]
```diff
@@ -92,48 +94,35 @@ export const CreateDirectoryDialog = ({
...
+            fullWidth={true}
             open={open}
             onClose={handleClose}
             aria-labelledby="dialog-title-delete"
             onKeyPress={handleKeyPressed}
         >
             <DialogTitle>{title}</DialogTitle>
             <DialogContent>
-                <InputLabel htmlFor="newName">{message}</InputLabel>
-                <Grid container spacing={3} className={classes.root}>
```
Comment (sBouzols) [inline src/components/dialogs/create-directory-dialog.js:103]: you removed the use of `classes.root` then remove its definition.

**Review by sBouzols — CHANGES_REQUESTED** [review]
Some code remarks. Dead code still here, to be removed.

File: src/components/dialogs/create-case-dialog.js (hunk 1 above)
Comment (klesaulnier) [reply src/components/dialogs/create-case-dialog.js]: fixed

File: src/translations/en.json (hunk 2 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/en.json (hunk 3 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/en.json (hunk 4 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/en.json (hunk 5 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/en.json (hunk 6 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/en.json (hunk 7 above)
Comment (klesaulnier) [reply src/translations/en.json]: fixed

File: src/translations/fr.json (hunk 8 above)
Comment (klesaulnier) [reply src/translations/fr.json]: fixed

File: src/translations/en.json [hunk 12]
```diff
@@ -81,7 +75,7 @@
     "deleteItemDialogTitle": "Delete {itemName}",
     "deleteMultipleItemsDialogTitle": "Delete {itemsCount} items",
     "renameDirectoryDialogTitle": "Rename the folder",
-    "directoryNameLabel": "Folder Name",
+    "directoryNameLabel": "Name",
```
Comment (sBouzols) [inline src/translations/en.json]: same as `nameProperty`

File: src/translations/fr.json [hunk 13]
```diff
@@ -80,7 +74,7 @@
     "deleteItemDialogTitle": "Supprimer {itemName}",
     "deleteMultipleItemsDialogTitle": "Supprimer {itemsCount} éléments",
     "renameDirectoryDialogTitle": "Renommer le dossier",
-    "directoryNameLabel": "Nom du dossier",
+    "directoryNameLabel": "Nom",
```
Comment (sBouzols) [inline src/translations/fr.json]: same

File: src/components/dialogs/create-directory-dialog.js (hunk 11 above)
Comment (sBouzols) [reply src/components/dialogs/create-directory-dialog.js:103]: And remove `message` props (+ JDoc + PropType)

File: src/components/dialogs/create-filter-dialog.js (hunk 10 above)
Comment (sBouzols) [reply src/components/dialogs/create-filter-dialog.js:247]: Still used in PopupInfo then ok

**Review by sBouzols — COMMENTED** [review]
Clean some `message={''}` in src/components/menus/directory-tree-contextual-menu.js
for AccessRightsDialog (unused), for CreateDirectoryDialog (unused and empty every time)

File: src/translations/fr.json (hunk 9 above)
Comment (sBouzols) [reply src/translations/fr.json]: optionnel in the US then ok

**Review by sBouzols — APPROVED** [review]
Code Review Ok
Tests done
Some remarks todo in another PR

**Pull request merged**